NitroxModel/DataStructures/GameLogic/Entities/Metadata/BulkheadDoorMetadata.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Runtime.Serialization;
+using BinaryPack.Attributes;
+
+namespace NitroxModel.DataStructures.GameLogic.Entities.Metadata
+{
+    [Serializable]
+    [DataContract]
+    public class BulkheadDoorMetadata : EntityMetadata
```

Comment:
You need to create test cases for the metadata in https://github.com/SubnauticaNitrox/Nitrox/blob/fabf9e1479966312b34aeb38eb332d8b62c3ec9f/Nitrox.Test/Server/Serialization/WorldPersistenceTest.cs#L330